mono/metadata/w32process-unix.c

```diff
@@ -2155,6 +2157,21 @@ ves_icall_System_Diagnostics_Process_ShellExecuteEx_internal (MonoW32ProcessStar
 			ret = FALSE;
 			goto done;
 		}
+
+		if (handler_needswait) {
+			gint32 exitcode;
+			MonoW32HandleWaitRet waitret;
+			waitret = process_wait (process_info->process_handle, MONO_INFINITE_WAIT, NULL);
+			if (ves_icall_Microsoft_Win32_NativeMethods_GetExitCodeProcess (process_info->process_handle, &exitcode)) {
+				if (exitcode != 0) {
+					ret = FALSE;
+					goto cleanup;
+				}
+			}
+			else
+				g_error ("%s: ves_icall_Microsoft_Win32_NativeMethods_GetExitCodeProcess failed", __func__);
```

Comment:
Please format it as follow:
```
if (!ves_icall_Microsoft_Win32_NativeMethods_GetExitCodeProcess (process_info->process_handle, &exitcode))
    g_error(...);
if (exitcode != 0) {
    ret = FALSE;
    goto cleanup;
}
```

Also, do we really need an assertion in this case?

Reply:
Nope, we don't :)